snap/local/helper-go/install.go

```diff
@@ -37,11 +38,25 @@ func installRTSPConfig() error {
 func install() {
 	log.SetComponentName("install")
 
-	if err := installUSBCameraConfig(); err != nil {
-		log.Fatalf("Error installing Device USB Camera files: %s", err)
+	// Install default config files for USB camera app only if no config provider is connected
+	isConnected, err := snapctl.IsConnected(usbCameraApp + "-config").Run()
+	if err != nil {
+		log.Fatalf("Error checking interface connection: %s", err)
+	}
+	if !isConnected {
+		if err := installUSBCameraConfig(); err != nil {
+			log.Fatalf("Error installing Device USB Camera files: %s", err)
+		}
 	}
 
-	if err := installRTSPConfig(); err != nil {
-		log.Fatalf("Error installing RTSP Simple Server config file: %s", err)
+	// Install default config files for RTSP server app only if no config provider is connected
+	isConnected, err = snapctl.IsConnected(rtspServerApp + "-config").Run()
```

Comment:
There is no such plug in the snap. I'm surprised that this isn't causing an installation error, but maybe it is actually skipped the copying of files.

Reply:
If a non-existent plug is passed to `snapctl is-connected`, the command will return exit status code 1:
```
$ snapctl is-connected non-exist-plug
error: error running snapctl: snap "edgexfoundry" has no plug or slot named "non-exist-plug"
$ echo $?
1
```

Reply:
I expect that would fall into [this](https://github.com/canonical/edgex-snap-hooks/blob/544c6823a1b007cf18e22cfa85d1ba12b0ffa2ba/snapctl/is-connected.go#L82) and return the error, but maybe there is a bug.

Reply:
Regardless, this PR should not check if a non-existing plug is connected.

Reply:
ok, I reverted install hook related changes

Reply:
I think you misunderstood my comments. Your code was checking two interfaces, one called `rtsp-simple-server-config` which doesn't exist, another called `device-usb-camera-config` which does exist and was correct. Please see the snapcraft.yaml file to know what interfaces are available for this snap, and need this fix: https://github.com/edgexfoundry/device-usb-camera/blob/96327e049a2d26e356251a4ee56d80af5c884d85/snap/snapcraft.yaml#L21